libraries/botbuilder-spell-check-middleware/src/spellcheck.ts

```diff
@@ -0,0 +1,55 @@
+import { Middleware, TurnContext, ActivityTypes } from "botbuilder";
+import * as WebRequest from 'web-request';
+/**
+ * @module botbuildercommunity/spellcheck
+ */
+
+export class SpellCheck implements Middleware {
+    public text: string;
+    public key: string;
+    constructor(public serviceKey: string) {
+        this.key = serviceKey;
+    }
+    public async onTurn(context: TurnContext, next: () => Promise<void>) {
+        if(context.activity.type === ActivityTypes.Message) {
+            this.text = context.activity.text;
+            let url = "https://api.cognitive.microsoft.com/bing/v7.0/spellcheck/?text=" + this.text + "&mode=spell"
+            try {
+                var re = await WebRequest.get(url, {
+                    headers : {
+                        'Content-Type' : 'application/x-www-form-urlencoded',
+                        'Content-Length' : 30,
```

Comment:
Why is the Content-Length hardcoded set to 30?

Reply:
Thanks for the heads up, will correct this!